recipes/nsis/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "2.46" %}
+
+package:
+  name: nsis
+  version: {{ version }}
+
+source:
+  fn: nsis-{{ version }}.tar.bz2
+  url: http://sourceforge.net/projects/nsis/files/NSIS%20{{ version.split(".")[0] }}/{{ version }}/nsis-{{ version }}-src.tar.bz2
+  sha256: f5f9e5e22505e44b25aea14fe17871c1ed324c1f3cc7a753ef591f76c9e8a1ae
+
+build:
+  number: 0
+  skip: true  # [unix]
```

Comment:
Is there any reason to build this with py34/35?  Last I checked, scons was py2 only.  I think the py34/35 builds are actually not producing good output.